src/Sarif.Multitool/PageCommand.cs

```diff
@@ -42,11 +43,7 @@ public void RunWithoutCatch(PageOptions options)
             if (options.Count < 0) { throw new ArgumentOutOfRangeException("count"); }
             if (!_fileSystem.FileExists(options.InputFilePath)) { throw new FileNotFoundException($"Input file \"{options.InputFilePath}\" not found."); }
 
-            if (options.Force == false && _fileSystem.FileExists(options.OutputFilePath))
-            {
-                Console.WriteLine($"Output file \"{options.OutputFilePath}\" already exists. Stopping.");
-                return;
-            }
+            DriverUtilities.VerifyOutputFileCanBeCreated(options.OutputFilePath, options.Force, _fileSystem);
```

Comment:
>VerifyOutputFileCanBeCreated [](start = 28, length = 28)

Since this is in a method named RunWithoutCatch and VerifyOutputFileCanBeCreated throws, does the user still get a nice error message?

Reply:
Yes and no. Yes, because the call to `RunWithoutCatch` is within a `try`/`catch` block, which prints the exception to the console. But no, because the `try/catch` block does print the whole exception, stack trace and all.

I've rearranged this and some other commands to give just the nice friendly error message when the error is due to user error.

---
In reply to: [311711199](https://github.com/microsoft/sarif-sdk/pull/1625#discussion_r311711199) [](ancestors = 311711199)